docs/zkasm/test_summary.csv

```diff
@@ -2,8 +2,8 @@ Suite path,Passing count,Total count,Total cycles
 <<<<<<< HEAD,,,
 =======,,,
 >>>>>>> main,,,
```

Comment:
Why do we have merge markers here...?

Reply:
My guess would be that they were committed accidentally. Removed them in f1b79ea.